robot/Cumulus/resources/GiftEntryPageObject.py

```diff
@@ -340,6 +361,19 @@ def verify_table_field_values(self,table,**kwargs):
         for field,value in kwargs.items():
             locator=npsp_lex_locators["gift_entry"]["table"].format(table,field,value)
             self.selenium.wait_until_page_contains_element(locator,error=f'{field} does not contain {value} in {table} table')
+    
+    def perform_action_on_datatable_row(self,name,action):
```

Comment:
I think we have a keyword that already does this, but it is called Select Template Action. I thought u wanted to edit data on the table when u asked me

Reply:
I tried using the keyword suggested. But does not work as-is. There could be element reference changes and logic changes needed.